src/App/AppWrapper.tsx

```diff
@@ -16,14 +17,15 @@ import App from './App.tsx';
 const environment = import.meta.env.MODE;
 const sentryOptions = {
     dsn: 'https://edd6b9c1494eb7fda6ee45590b80bcee@o4504839079002112.ingest.us.sentry.io/4507979991285760',
-    integrations: [Sentry.captureConsoleIntegration({ levels: ['warn', 'error'] })],
+    integrations: [Sentry.captureConsoleIntegration({ levels: ['warn', 'error'] }), Sentry.extraErrorDataIntegration()],
```

Comment:
Couldn't we just turn this down to `'error'` and remove `'warn'` and then Sentry wouldn't get warnings, or am I oversimplifying it?

Reply:
we don't actually have that many `console.warn`s, none we've added specifically, usually they come from the webview console and that's where a couple of the webgl warnings come from so figured it'd be good to leave in... 

when i said here 

> added ignore for all the "Already getting ...." warnings from BE which are being logged as errors clogging up sentry

the "warn"ings are warns from BE which come to the FE as errors in the catch, separate from `console.warn()` warnings 😅 

those three warnings/errors are the main ones clogging up sentry, and i don't think we need them logged really... certainly not in sentry (but didn't want to remove the source logs entirely:D)

Reply:
hmm, I guess i'm okay with it for now. Would be nice if the BE and FE could negotiate the difference between a warning and an error by type instead of by message but that can be a future enhancement.

Reply:
think the tauri v2 sentry plugin  will help a bunch;)